test/JDBC/expected/indexproperty-vu-prepare.out

```diff
@@ -0,0 +1,26 @@
+use master 
+go
+
+-- should auto-name the primary key index as t1_pkey
+create table t1(c1 int PRIMARY KEY (c1))
```

Comment:
Object name should be prefixed with test file name.. for example, indexproperty-vu-prepare-t1

Reply:
Done.